File: TP1/game.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>

int main() {
    int secret, guess;

    // initialisation de la graine (seed)
    srand(time(NULL));

    secret = rand() % 10 + 1;

    printf("J'ai choisi un nombre entre 1 et 10\n");
    printf("Votre proposition ? ");
    scanf("%d", &guess);
    
    if (guess < secret) {
        printf("Plus grand !\n");
    } else if (guess > secret) {
        printf("Plus petit !\n");
    } else {
        printf("Trouvé !\n");
    }
    printf("C'était %d\n", secret);
    exit(0);
}
    

File: TP3/game.py
#!/usr/bin/env python3

from random import randint

secret = randint(1, 100)

found = False

n = 0

print("J'ai choisi un nombre entre 1 et 100.")

while not found:
    guess = int(input("Votre choix : "));
    n += 1
    if guess < secret:
        print("Plus grand.")
    elif guess > secret:
        print("Plus petit.")
    else:
        found = True;
        print(f"Trouvé ! En {n} coups.");


